bot/controllers/tosWelcome.js

```diff
@@ -0,0 +1,28 @@
+const BaseController = require('../baseController.js');
+const Command = require('../baseCommand.js');
+
+class TosController extends BaseController {
+  constructor(message) {
+    // Call BaseController constructor
+    super(message);
+    // Aliasing 'this' as controller to allow for binding in actions
+    const controller = this;
+    // Array of all commands, see baseCommand.js for prototype
+    this.commands = [
+      new Command(
+        '!tos',
+        '!tos',
+        'Resend Terms',
+        'Send Terms Copy',
```

Comment:
Even though we are short handing the the ToS down to a simple one liner, the help command should at least specify that it's the Terms of Service. 'Resend Terms' and 'Send Terms copy' would be acceptable for our internal use, but consider that this help message is user facing... and [users will surprise you with how they interact with your products](https://twitter.com/designuxui/status/576432203560685568?lang=en).